serverless/src/shared/toKeywordJson.js

```diff
@@ -0,0 +1,226 @@
+import { castArray } from 'lodash'
+
+import { buildFullPath } from './buildFullPath'
+import { cleanupJsonObject } from './cleanupJsonObject'
+import { createChangeNoteItem } from './createChangeNoteItem'
+import { getNumberOfCmrCollections } from './getNumberOfCmrCollections'
+import { getVersionMetadata } from './getVersionMetadata'
+/**
+ * Processes alternative labels (altLabels) from GCMD format.
+ * @param {Object|Array} altLabels - The altLabels to process.
+ * @returns {Array} An array of processed altLabels.
+ *
+ * @example
+ * const altLabels = [
+ *   { '@gcmd:category': 'primary', '@gcmd:text': 'Label 1', '@xml:lang': 'en' },
+ *   { '@gcmd:text': 'Label 2', '@xml:lang': 'fr' }
+ * ];
+ * const processed = processAltLabels(altLabels);
+ * // Result: [
+ * //   { category: 'primary', text: 'Label 1', languageCode: 'en' },
+ * //   { text: 'Label 2', languageCode: 'fr' }
+ * // ]
+ */
+export const processAltLabels = (altLabels) => {
```

Comment:
If these are helper methods for toKeywordJson and you don't want them to standalone, move them inside of toKeywordJson.

Reply:
Fixed